### PR TITLE
adapt some firefox regression tests

### PR DESCRIPTION
**Pull request by mitiao**

### CODE REVIEW

**Comment by okurz** [comment]
Please see https://github.com/os-autoinst/os-autoinst#rules-for-commits especially http://chris.beams.io/posts/git-commit/

Your commit message should describe a bit better what your change does, why it is good to have it and not describe the "how". Also start the subject line with capital letter.

**Comment by mitiao** [comment]
add detailed comment, thanks for reminding

File: tests/x11regressions/firefox/sle12/firefox_flashplayer.pm [hunk 1]
```diff
@@ -28,12 +28,17 @@ sub run() {
...
 
     send_key "pgdn";
-    assert_screen('firefox-flashplayer-verify', 25);
-
-    send_key "esc";
-    send_key "alt-d";
-    type_string "https://www.youtube.com/watch?v=Z4j5rJQMdOU\n";
-    assert_screen('firefox-flashplayer-video_loaded', 45);
+    # flashplayer dropped since sled12 sp2
+    while (check_screen([qw/firefox-flashplayer-dropped firefox-flashplayer-verify/])) {
```
Comment (coolo) [inline tests/x11regressions/firefox/sle12/firefox_flashplayer.pm]: hmm, don't you rather assert that you see one or the other?

Comment (mitiao) [inline tests/x11regressions/firefox/sle12/firefox_flashplayer.pm]: I think I don't get what you exactly mean :(

on sled12 sp2, flashplayer dropped and page shows like this:
https://openqa.suse.de/tests/279168/modules/firefox_flashplayer/steps/9

and on pre-sp2, flashplayer exist and page shows its version info like this:
http://147.2.207.208/tests/797/modules/firefox_flashplayer/steps/7

Comment (coolo) [inline tests/x11regressions/firefox/sle12/firefox_flashplayer.pm]: yes, but now you check_screen which might miss. What you really should do IMO is *assert* that one of the screens appears - i.e. fail if not. Just change check_screen to assert_screen should do that

Comment (mitiao) [inline tests/x11regressions/firefox/sle12/firefox_flashplayer.pm]: Ah... I see.

**Pull request closed without merging**